accounts/admin.py

```diff
@@ -5,7 +5,53 @@
 
 
 class UserAdmin(BaseUserAdmin):
+    list_display = (
+        "email",
+        "is_staff",
+        "is_active",
+    )
     ordering = ("email",)
+    search_fields = ("email",)
+    list_filter = (
+        "is_staff",
+        "is_active",
+    )
+
+    # Fields used on the view/edit user page
+    fieldsets = (
+        ("Personal info", {"fields": ("email", "first_name", "last_name", "password")}),
+        ("Important dates", {"fields": ("last_login", "date_joined")}),
+        (
+            "Permissions",
+            {
+                "fields": (
+                    "is_active",
+                    "is_staff",
+                    "is_superuser",
+                    "groups",
+                    "user_permissions",
+                )
+            },
+        ),
+    )
+
+    # Fields used on the add user page
+    add_fieldsets = (
+        (
+            None,
+            {
+                "fields": (
+                    "first_name",
+                    "last_name",
+                    "email",
+                    "password1",
+                    "password2",
```

Comment:
Is there anywhere we're checking `password1 == password2` or is this done under the hood 😮?

Reply:
This is done under the hood! You get this error if they don't match:

<img width="1056" alt="Screen Shot 2021-02-06 at 7 04 39 PM" src="https://user-images.githubusercontent.com/37346399/107132573-2b00c100-68ae-11eb-97d0-032c9f909b95.png">

Reply:
django is too strong 💪